File: src/indexer/test_relationships/python/subpackage/module2.py
"""Module 2 for testing Python relationship extraction."""

# Standard library import
import datetime

# Relative import (going up one level)
from .. import module1


def function2():
    """Test function in module2."""
    print("Function 2 from module2")
    print(f"Current time: {datetime.datetime.now()}")

File: src/indexer/test_relationships/typescript/utils/module2.ts
/**
 * Module 2 for testing TypeScript relationship extraction.
 */

// Import from parent directory
import { function1 } from '../module1';

/**
 * Example function
 */
export function function2(): void {
  console.log('Function 2 from module2');
  console.log(`Current time: ${new Date().toISOString()}`);
}

/**
 * Side effect when imported
 */
console.log('Module 2 loaded');
